app/controllers/service_controller.rb

```diff
@@ -379,7 +194,7 @@ def set_right_cell_vars(action)
       action = "dialog_form_button_pressed"
     when "ownership"
       partial = "shared/views/ownership"
-      header = _("Set Ownership for Service")
+      header = _("Set Ownership for Service1")
```

Comment:
@jeffibm @Fryguy Is this a typo? If it is I fixed it in pr: https://github.com/ManageIQ/manageiq-ui-classic/pull/8396. If it's not a typo feel free to close my pr.